### ISSUE TITLE
Variable stars experiment links

### ISSUE THREAD

**Issue by fserena** [GLORIA-project/users-hub-interface#4]
As Filip proposed, I am going to include a set of links and a banner for the Variable stars experiment (Luiza).

![variable-stars](https://f.cloud.github.com/assets/2788595/2041442/2fc71dd0-89c5-11e3-9eed-44796020e904.PNG)

I need a better short description. Please Filip, post here the one you want.

**Comment by zarnecki** [comment]
I will try to prepare both the graphic and the text for tomorrow. Thanks a lot for your help.

**Comment by zarnecki** [comment]
![Uploading luiza-logo.png . . .]()
What about : 

Variable stars
Hunt for variable stars in the data from wide field sky observations. 

![Uploading luiza-logo.png . . .]()

**Comment by zarnecki** [comment]
![luiza-logo](https://f.cloud.github.com/assets/6538962/2045482/42aa495c-89f9-11e3-840f-f2070563ef59.png)

Variable stars
Hunt for variable stars in the data from wide field sky observations.

(sorry for posting again, but the image was not attached before)

**Comment by zarnecki** [comment]
I have also updated the logo on the main experiment page:
http://193.0.87.199/luiza/interface/pi-lc-server.html

If you are going to redirect user to our server please use the link above and not the one which is put in test page now (.../pi-lc-intro.html), as it does not include banner with the logo.

**Comment by fserena** [comment]
Updated!

**Comment by zarnecki** [comment]
Hi Fernando,

Thanks a lot. But where can I see it?
Currently at the users page

http://users.gloria-project.eu/#/

I still see old graphics and wrong link.
The link should point to:
   http://193.0.87.199/luiza/interface/pi-lc-server.html
(if opened in the new window/tab).

By the way: I just updated the interface so the period fitting works much
     better now...

Regards
  Filip

On Mon, 17 Feb 2014, Fernando Serena wrote:

> Closed #4.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/GLORIA-project/UsersHub/issues/4

---

```
  prof. dr hab. A.F.Zarnecki
     Division of Particles
  and Fundamental Interactions
Institute of Experimental Physics
```

Faculty of Physics, University of Warsaw
      Hoza 69, 00-681 Warszawa
              POLAND

   tel: (+4822) 5532365      http://www.fuw.edu.pl/~zarnecki
   fax: (+4822) 5532106      e-mail:  zarnecki@fuw.edu.pl

---

**Comment by fserena** [comment]
Hi Filip, you may need to remove your browser cache. The link is wrong, sorry, I will update it this afternoon. Thanks.
